kuzzleio/documentation
# Update seo

## Apply to the prod documentation the latest SEO upgrades

### How should this be manually tested?

  - Step 1 : Apply in prod 
  - Step 2 : run semrush analysis 
  - Step 3 :  ask google for reindexation 
  - Step 4 : check if v2 links pops up above v1 links
